File: RemoteMonitor Desktop/src/main/java/components/Component.java
package components;

import oshi.SystemInfo;

public class Component {
    protected String componentName;
    protected String componentType;
    protected long lastUpdated;
    protected double temperature;
    protected double usage;
    protected transient SystemInfo si;

    public Component()
    {
        si = new SystemInfo();
    }
    public double getTemperature(){
        return temperature;
    }

    public String getComponentName() {
        return componentName;
    }

    public double getUsage() {
        return usage;
    }

    public long getLastUpdated() {
        return lastUpdated;
    }
}

File: RemoteMonitor Mobile/RemoteMonitor/RemoteMonitor/RemoteMonitor/Components/Component.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RemoteMonitor.Components
{
    public class Component
    {
        public string componentName;
        public string componentType;
        public long lastUpdated;
        public double temperature;
        public double usage;

        public double getTemperature()
        {
            return temperature;
        }

        public String getComponentName()
        {
            return componentName;
        }

        public double getUsage()
        {
            return usage;
        }

        public long getLastUpdated()
        {
            return lastUpdated;
        }
    }
}
